src/components/container/SurveyTaskList/SurveyTaskList.css

```diff
@@ -12,6 +16,10 @@
     padding-bottom: 0;
 }
 
+.mdhui-card-title {
+    color: rgba(60, 60, 67, 0.60)!important;
```

Comment:
Few issues with this:
- It will override the card title style for the entire library
- We should use variables for colors
- We should absolutely avoid using !important
- We should just make this a customization option

Reply:
I can tackle this as part of the color customization work i'm doing since i'm about to touch this